tests/testthat/test-validate_r.R

```diff
@@ -0,0 +1,228 @@
+test_that("Custom R compotnent with multiple return()s fails ", {
```

Comment:
Typo in the test description: 'compotnent' should be 'component'
```suggestion
test_that("Custom R component with multiple return()s fails ", {
```